fastly/custom_tls_certificate.go

```diff
@@ -0,0 +1,195 @@
+package fastly
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/google/jsonapi"
+)
+
+// CustomCertificate represents a custom certificate. Uses common TLSDomain type from BulkCertificate
+type CustomCertificate struct {
+	ID                 string       `jsonapi:"primary,tls_certificate"`
+	CreatedAt          *time.Time   `jsonapi:"attr,created_at,iso8601"`
```

Comment:
To follow convention in the project do you mind moving this metadata field to the end of the struct.

Reply:
Done!